qiskit_nature/operators/second_quantization/fermionic_op.py

```diff
@@ -766,3 +769,10 @@ def one(cls, register_length: int) -> FermionicOp:
             The unity-operator of the given length.
         """
         return FermionicOp(("", 1.0), register_length=register_length, display_format="sparse")
+
+    def is_hermitian(self, atol: Optional[float] = None) -> bool:
+        """Checks whether the operator is hermitian"""
+        if atol is None:
+            atol = self.atol
+        diff = (self - self.adjoint()).normal_ordered().simplify(atol=atol)
```

Comment:
Can you please provide an example where the previous (or current implementation without calling `normal_ordered()`) fails?